bundleserver/src/main/java/net/discdd/server/applicationdatamanager/AduStores.java

```diff
@@ -12,8 +12,8 @@ public class AduStores {
     private final StoreADUs sendADUsStorage;
 
     public AduStores(@Value("${bundle-server.bundle-store-root}") Path rootDataDir) {
-        this.sendADUsStorage = new StoreADUs(rootDataDir.resolve("send"), true);
-        this.receiveADUsStorage = new StoreADUs(rootDataDir.resolve("receive"), false);
+        this.sendADUsStorage = new StoreADUs(rootDataDir.resolve("send").toFile().toPath(), true);
```

Comment:
Without this BundleServer has build issues, is it working for others?

Reply:
i think you need to rebase to the latest.

Reply:
you you don't need this. toFile().toPath() is a noop since it was already a path.